src/main/java/nextstep/subway/fare/domain/Fare.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.subway.fare.domain;
+
+import nextstep.subway.auth.domain.LoginMember;
+
+public class Fare {
+    private static final int DEFAULT_FARE = 1_250;
+
+    private final int value;
+
+    private Fare(int value) {
+        this.value = value;
+    }
+
+    public static Fare of(final LoginMember loginMember, final int lineExtraCharge, final int distance) {
+        final int totalCharge = totalCharge(lineExtraCharge, distance);
+        if (isNonMember(loginMember)) {
```

Comment:
별도로 아래에서 메서드를 만들지말고 바로 호출해도 되지 않을까해요 :D
```suggestion
        if (loginMember.isNonMember()) {
```